.github/workflows/_integration_tests.yaml

```
@@ -51,11 +51,7 @@ jobs:
             test: postgres
           - name: feature-gates
             test: dbless
-            feature_gates: "GatewayAlpha=true"
-          - name: dbless-knative
-            test: dbless.knative
-          - name: postgres-knative
-            test: postgres.knative
+            feature_gates: "GatewayAlpha=true,CombinedRoutes=false"
```

```suggestion
            feature_gates: "GatewayAlpha=true"
```
we don't have it anymore
- https://github.com/Kong/kubernetes-ingress-controller/pull/4749

---

Alright then this becomes the same as `dbless-gateway-alpha` which means we can get rid of this